lib/conjur/cli.rb

```diff
@@ -91,6 +93,27 @@ def init!
         load_plugins
         commands_from 'conjur/command'
       end
+
+      def command_version_compatible? command
```

Comment:
This feels kind of un-ruby-ish. Maybe consider something like this:

```
  def appliance_version
    Conjur::API.service_version 'appliance'
  rescue
    nil
  end

  def command_version_compatible? command
    !command.instance_variable_defined?(:@conjur_min_version) ||
      appliance_version.try { |v| v < command.instance_variable_get(:@conjur_min_version) }
  end
```
which takes advantage of the fact that `nil` and `false` are equivalent in boolean tests. (Note that I haven't actually tested this, so it might have typos.)